src/multiplexed-session.ts

```diff
@@ -197,7 +195,10 @@ export class MultiplexedSession
    *
    */
   async _acquire(): Promise<Session | null> {
+    const span = getActiveOrNoopSpan();
+    span.addEvent('Acquiring multiplexed session');
```

Comment:
have added these events for multiplexed session similar to we had for regular sessions

Reply:
nit: I feel these events are not required. For regular session it was required as there was a wait if the session is not available. We already have few events in `_getSession` for mux sessions